Modix.Bot/Modules/FunModule.cs

```diff
@@ -134,9 +134,15 @@ public async Task OwoifyAsync([Remainder][Summary("The message to owoify.")] str
             owoMessage = Regex.Replace(owoMessage, "N([aeiou])", "Ny$1");
             owoMessage = Regex.Replace(owoMessage, "N([AEIOU])", "Ny$1");
             owoMessage = Regex.Replace(owoMessage, "ove", "uv");
-            owoMessage = Regex.Replace(owoMessage, "\\!+", " " + _owoFaces[new Random().Next(0, _owoFaces.Length - 1)] + " ");
-
-            await ReplyAsync(owoMessage);
+            owoMessage = Regex.Replace(owoMessage, "(?<!\\@)\\!+", " " + _owoFaces[new Random().Next(_owoFaces.Length)] + " ");
+
+            await ReplyAsync(embed: new EmbedBuilder()
+                .WithDescription(owoMessage)
+                .WithAuthor(Context.Message.Author)
+                .WithColor(Color.Blue)
+                .WithTimestamp(DateTime.UtcNow)
```

Comment:
I think Discord.Net exposes a `WithCurrentTimestamp()` to do this
```suggestion
                .WithCurrentTimestamp()
```